app-android/src/androidTest/java/com/ustadmobile/port/android/screen/LeavingReasonListScreen.kt

```diff
@@ -0,0 +1,52 @@
+package com.ustadmobile.port.android.screen
+
+import android.view.View
+import com.agoda.kakao.common.views.KView
+import com.agoda.kakao.image.KImageView
+import com.agoda.kakao.recycler.KRecyclerItem
+import com.agoda.kakao.recycler.KRecyclerView
+import com.agoda.kakao.text.KTextView
+import com.kaspersky.kaspresso.screens.KScreen
+import com.toughra.ustadmobile.R
+import com.ustadmobile.port.android.view.LeavingReasonListFragment
+import org.hamcrest.Matcher
+
+object LeavingReasonListScreen : KScreen<LeavingReasonListScreen>() {
+
+
+    override val layoutId: Int?
+        get() = R.layout.fragment_list
+    override val viewClass: Class<*>?
+        get() = LeavingReasonListFragment::class.java
+
+    val recycler: KRecyclerView = KRecyclerView({
+        withId(R.id.fragment_list_recyclerview)
+    }, itemTypeBuilder = {
+        itemType(::LeavingReason)
+        itemType(::SortOption)
+    })
+
+
+    val sortList: KRecyclerView = KRecyclerView({
+        withId(R.id.fragment_sort_order_list)
+    }, itemTypeBuilder = {
+        itemType(::Sort)
+    })
+
+
+    class LeavingReason(parent: Matcher<View>) : KRecyclerItem<LeavingReason>(parent) {
+        val title: KTextView = KTextView(parent) { withId(R.id.line1_text) }
+    }
+
+    class SortOption(parent: Matcher<View>) : KRecyclerItem<SortOption>(parent) {
```

Comment:
If SortOption is something that we have on every list screen, would it make sense to Create KListScreen and then have those in the parent class?